### PR TITLE
added vissually-hidden text for table-headers

### PR DESCRIPTION
**Pull request by Leeroyc720**

### CODE REVIEW

File: app/views/propertyrepresentation/appoint/appointAgentProperties.scala.html [hunk 1]
```diff
@@ -143,8 +143,8 @@ <h2 class="heading-secondary margin-top-30">@messages("dashboard.managePropertie
                          <th aria-sort="none">@messages("propertyRepresentation.appointAgents.table.select")</th>
                          <th class="search-sort-table-header" aria-sort="@ariaSortOrder(params.sortorder , params.sortfield == ADDRESS)">@sortingControl(ADDRESS, backLinkAsString)</th>
                          <th class="search-sort-table-header" aria-sort="@ariaSortOrder(params.sortorder , params.sortfield == AGENT)" colspan="2">@sortingControl(AGENT, backLinkAsString)</th>
-                         <th scope="col" width="30px" role="columnheader">
-                     </tr>
+                         <th scope="col" width="30px" role="columnheader"><caption class="visuallyhidden">@messages("propertyRepresentation.appointAgentProperties.title")</caption>
```
Comment (PeterPerhac) [inline app/views/propertyrepresentation/appoint/appointAgentProperties.scala.html:146]: I think this is missing closing tag </th>

Comment (PeterPerhac) [reply app/views/propertyrepresentation/appoint/appointAgentProperties.scala.html:146]: `</th>`

File: app/views/propertyrepresentation/revokeAgentProperties.scala.html [hunk 2]
```diff
@@ -115,7 +115,7 @@ <h2 class="heading-secondary margin-top-30">@messages("dashboard.managePropertie
             <tr role="row">
                 <th aria-sort="none">@messages("propertyRepresentation.appointAgents.table.select")</th>
                 <th class="@sortAddress search-sort-table-header" aria-sort="@ariaSortAddress">
-                <th scope="col" width="30px" role="columnheader">
+                <th scope="col" width="30px" role="columnheader"><caption class="visuallyhidden">@messages("propertyRepresentation.removeAgentProperties.heading")</caption>
```
Comment (PeterPerhac) [inline app/views/propertyrepresentation/revokeAgentProperties.scala.html:118]: this looks fine, hopefully there's a closing `</th>` tag further down

**Comment by darrell-plant** [comment]
👍 lgtdp

**Comment by reddy-yattapu** [comment]
:+1:

**Pull request merged**